frontend/src/components/Post/PostImage.js

```diff
@@ -0,0 +1,12 @@
+import '../../config';
+import React from "react";
+
+import "./PostImage.scss";
+
+export default function PostImage(pictureId) {
```

Comment:
Why not `props`? `pictureId.pictureId` looks a little weird no?

Reply:
Does it? It's the only property I need here so I thought it more fitting to give it an appropriate name

Reply:
I don't see the point of calling the `props` as `pictureId` to make it more descriptive because you'd need to write `props.pictureId` anyway. I would even say it makes it even less clear because calling it `props` makes it easier to tell that the pictureId is passed from another component. If you change it it's hard to tell, and there's also no guarantee that it's going to be the only needed property in the future.

Reply:
Changes have been made to reflect this 😄